sphinx/util/pycompat.py

```diff
@@ -103,3 +103,33 @@ def execfile_(filepath, _globals):
         else:
             raise
     exec_(code, _globals)
+
+# ------------------------------------------------------------------------------
+# Internal module backwards-compatibility
+
+import functools
+import warnings
+
+def _deprecated(func):
+    @functools.wraps(func)
+    def wrapped(*args, **kwargs):
+        warnings.warn("sphinx.util.pycompat.{0.__name__} is deprecated, "
+                      "please use the standard library version.".format(func),
+                      DeprecationWarning, stacklevel=2)
```

Comment:
Added to the warning text.  I briefly thought about making the version configurable on a per-deprecation basis, but since we only have this one set of deprecations right now…